it/controllers/benefits/accommodation/NonQualifyingRelocationBenefitsControllerISpec.scala

```diff
@@ -34,7 +34,7 @@ import utils.PageUrls.{checkYourBenefitsUrl, fullUrl, nonQualifyingRelocationBen
 
 class NonQualifyingRelocationBenefitsControllerISpec extends IntegrationTest with ViewHelpers with EmploymentDatabaseHelper {
 
-  private val taxYearEOY: Int = 2021
+  private val taxYearEOY: Int = taxYear - 1
```

Comment:
If we use tax year from IntegrationText with my suggestion of adding +1, the above will produce failures. I think we can subtract 2, i.e. taxYear - 2.
Alternatively we can fix IntegrationTestClock and can use variables from there.